test/pyca/cryptography.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/sh
```

Comment:
(`ossl_shim` could do the same kind of move...  but that was made pre `data_file` and we haven't got around to it yet...)